### DISCUSSION TITLE
Is it possible to open Windows Bluetooth settings?

### DISCUSSION THREAD

**Discussion by thecouchcoder** [Flow-Launcher/Flow.Launcher#2815] [Q&A]
I'd like to type "bluetooth" or "blt" and either open the Windows settings page for bluetooth or be able to manage it directly in flow launcher.  Is it possible right now?

**Comment by jjw24** [comment]
There should be a result for Bluetooth settings page if you use the action keyword `s` and search. Otherwise the Plugin Store iirc has Bluetooth related plugins.

**Comment by thecouchcoder** [comment]
@jjw24 Thank you! I didn't realize I needed to type `s` first.

**Comment by YashDoesCode** [comment]
> I'd like to type "bluetooth" or "blt" and either open the Windows settings page for bluetooth or be able to manage it directly in flow launcher. Is it possible right now?

Yes ! You can setup a Hotkey (let's suppose Alt + B) which would autmatically type "Bluetooth" in the search field of Flow Launcher.

Now, you just need to click enter and it would open the Bluetooth settings windows on Windows Operating System.

For this to work, make sure that you have "Settings" names plugin installed on your Flow (By Default : Settings plugin is installed)

I hope that would resolve your problem!
